### PR TITLE
[16.0][MIG] web_remember_tree_column_width: Migration to 16.0

### PR DESCRIPTION
**Pull request by cuongnmtm**

### CODE REVIEW

**Review by kikopeiro — APPROVED** [review]
:+1:  LGTM (without code review)

**Review by quannguyen1560 — APPROVED** [review]
Functional Review: LGTM (without code review)

**Review by CoraVanga — APPROVED** [review]
LGTM

File: web_remember_tree_column_width/static/src/js/list_renderer.esm.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+patch(ListRenderer.prototype, "web_remember_tree_column_width.ListRenderer", {
+    /**
+     * @override
+     */
+    computeColumnWidthsFromContent() {
+        const columnWidths = this._super.apply(this, arguments);
+        const table = this.tableRef.el;
+        const thElements = [...table.querySelectorAll("thead th:not(.o_list_button)")];
+        const self = this;
+        thElements.forEach(function (el, elIndex) {
```
Comment (Leonidas-VII) [inline web_remember_tree_column_width/static/src/js/list_renderer.esm.js]: For this line, we could use the lambda function, we can improve like this

`thElements.forEach((el, elIndex) => {`

Then no need to assign self=this anymore, we can re-use `this`

File: web_remember_tree_column_width/static/src/js/list_renderer.esm.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            const fieldName = $(el).data("name");
+            if (self.props.list.resModel && fieldName && browser.localStorage) {
+                const storedWidth = browser.localStorage.getItem(
+                    "odoo.columnWidth." + self.props.list.resModel + "." + fieldName
```
Comment (Leonidas-VII) [inline web_remember_tree_column_width/static/src/js/list_renderer.esm.js]: For this line, we can use `template literals` here for more simple

image

Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#syntax

**Review by Leonidas-VII — CHANGES_REQUESTED** [review]

**Comment by minhthie** [comment]
functional test: lgtm

**Comment by liebana** [comment]
@cuongnmtm were you able to attend @Leonidas-VII comments?

From my side, functional revision LGTM without those requests

**Comment by cuongnmtm** [comment]
> @cuongnmtm were you able to attend @Leonidas-VII comments?
> 
> From my side, functional revision LGTM without those requests

@liebana I will attend to those comments this week.
Those suggested changes are for code refactoring which uses new Javascript ES6 features to make the code look better.

**Comment by dsolanki-initos** [comment]
Hi @cuongnmtm, Could you please adjust your commit as per the OCA guidelines? Refer https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0.

**Review by Leonidas-VII — APPROVED** [review]
LGTM 😄

**Review by StefanRijnhart — APPROVED** [review]
Thanks! (Code review + functional test)

**Pull request merged**